server/main.go

```diff
@@ -54,6 +54,11 @@ func main() {
 	if err != nil {
 		glog.Fatalln("Failed to new genesis config.", zap.Error(err))
 	}
+	// set genesis timestamp
+	genesis.SetGenesisTimestamp(genesisCfg.Timestamp)
```

Comment:
here 
```
genesisCfg, err := genesis.New()
```
takes timestamp input from config/yaml file, so should call SetGenesisTimestamp() here